### PR TITLE
Make warrior upset when losing points

### LINKED ISSUE #104: Warrior should be upset when losing points
Currently, the warrior [is proud of earning points](https://github.com/olistic/warriorjs/blob/fbc5c10c30124f45bb876a6ef2a5a6d59ec36436/packages/warriorjs-core/src/Warrior.js#L37-L40), but he or she should also be upset for losing points.

### PR DESCRIPTION
**Pull request by skywalker212**
Closes #104

### CODE REVIEW

**Comment by skywalker212** [comment]
@olistic ya sure, but I don't know how to add a unit test, can you tell me how to add one?

**Comment by olistic** [comment]
@skywalker212 Sure. In this particular case, it's super easy. Just look at the earning points test in [`Warrior.test.js`](https://github.com/olistic/warriorjs/blob/fbc5c10c30124f45bb876a6ef2a5a6d59ec36436/packages/warriorjs-core/src/Warrior.test.js#L44-L47) and base yours on that one!

**Comment by skywalker212** [comment]
@olistic I have added the unit test, please review.

File: packages/warriorjs-core/src/Warrior.test.js [hunk 1]
```diff
@@ -45,4 +45,10 @@ describe('Warrior', () => {
     warrior.earnPoints(5);
     expect(warrior.say).toHaveBeenCalledWith('earns 5 points');
   });
+
+  test('is upset for losing points', () => {
+    warrior.losePoints(5);
+    expect(warrior.say).toHaveBeenCalledWith('loses 5 points');
+  });
+
```
Comment (olistic) [inline packages/warriorjs-core/src/Warrior.test.js]: @skywalker212 Linting is complaining because of this extra line break.

**Comment by skywalker212** [comment]
@olistic I have made the changes, please review.

**Comment by skywalker212** [comment]
@olistic Thanks for guiding through problems!

**Pull request merged**